feat: update details for http request headers (#1189)
edgexfoundry/edgex-docs: docs_src/microservices/application/BuiltIn.md

    Export the Events to  different endpoints base on their device name              
    `Url="http://myhost.com/edgex-events/{devicename}"` 

#### HTTP Request Header Parameters

| Method                                                                                                                           | Description                                                                                                                                                                                                                                                                                          |
|------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | SetHttpRequestHeaders(httpRequestHeaders map[string]string) | This function returns a `HTTPSender` instance initialized with the passed in HTTP request header parameters  |      

Reviewer: ```suggestion
| SetHttpRequestHeaders(httpRequestHeaders map[string]string) | This function sets the with the request header parameters which will be passed in HTTP request  |      
```
